osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHitStrong.cs

```diff
@@ -83,7 +83,7 @@ public override bool OnPressed(TaikoAction action)
                 return false;
 
             // Assume the intention was to hit the strong hit with both keys only if the first key is still being held down
-            return firstKeyHeld && UpdateJudgement(true);
+            return firstKeyHeld && Judgements.Last().IsHit;

```

Comment:
I think `UpdateJudgement` still needs to be called here, right?

Reply:
Yeah this is the line that actually handles the second hit.